how_to_use_package.md

```diff
@@ -0,0 +1,47 @@
+# 🚀 Quick Start Guide for matplotlib_visual
+--- 
+
+## **🔧Installations:**
+```bash
+pip install package
+```
+> **Note:** The `matplotlib-visual` package has been implemented by [Project Manager](https://github.com/this-is-yaash) and has details specific to him, you are sugested to follow the steps above and fork the repository and upload a package.
+
+## **⏱️Execution:**
+1. Create a Python file with **.py** extension and enter any of the code snippets below.
+2. Or, open a terminal and run Python interactively:
+```bash
+python
+```
+
+### 📊Features of this package:
+- **📍Displaying source code** in the console output.
+```python
+    from package.models import display
+    display("<filename-without-extension>")
+```
+ *`models` method displays source code in the console.*  
```

Comment:
Thanks for pointing it out, it might cause the user to run the command first then understand it, because the user might not read the entire content and just run the command.
I have fixed this issue as instructed, please check once.